integrations/cross_platform/yocto.rst

```diff
@@ -0,0 +1,213 @@
+.. _yocto_integration:
+
+
+|yocto_logo| Yocto
+__________________
+
+The `Yocto Project`_ is an open-source project that delivers a set of tools that create operating system images for embedded Linux systems.
+The Yocto Project tools are based on the `OpenEmbedded`_ project, which uses the BitBake build tool, to construct complete Linux images.
+
+Brief introduction to Yocto
+===========================
+
+Yocto supports several Linux host distributions and it also provides a way to install the correct version of these tools by either
+downloading a buildtools-tarball or building one on a supported machine. This allows virtually any Linux distribution to be able to run
+Yocto, and also makes sure that it will be possible to replicate your Yocto build system in the future. The Yocto Project build system also
+isolates itself from the host distribution's C library, which makes it possible to share build caches between different distributions and
+also helps in future-proofing the build system.
+
+Creating Conan packages with Yocto's SDK toolchain
+==================================================
+
+Yocto SDKs are completely self-contained. The binaries are linked against their own copy of `libc`, which results in no dependencies on the
+target system.
+
+Then, you can start creating Conan packages setting up the environment of the Yocto SDK and running a :command:`conan create` command
+with a suitable profile with the specific architecture of the toolchain.
+
+For example, creating packages for `arch=armv8`:
+
+The profile will be:
+
+.. code-block:: text
+   :caption: *armv8*
+
+    [settings]
+    os_build=Linux
+    arch_build=x86_64
+    os=Linux
+    arch=armv8
+    compiler=gcc
+    compiler.version=8
+    compiler.libcxx=libstdc++11
+    build_type=Release
+
+Activate the SDK environment and execute the create command.
+
+.. code-block:: bash
+
+    $ source oe-environment-setup-aarch64-poky-linux
+    $ conan create . user/channel --profile armv8
+
+This will generate the packages using the Yocto toolchain. Now you can upload the binaries to an Artifactory server so share and reuse in
+your Yocto builds.
+
+.. note::
```

Comment:
This shouldn't be a "note". Maybe we should create another section before this one to `Setting up a Yocto SDK` or similar. Mention there how to generate the SDK from your project and the idea of getting available in the SDK all the libraries/tools that you specify.